prog13.rb

```diff
@@ -0,0 +1,13 @@
+# Rewrite the factorial method using ranges. Your script should take a number from the command line 
+# and output the result on standard output
+# [input]
+# 0
+# [output]
+# 1
+
+num=ARGV[0].to_i
```

Comment:
This program will work absolutely fine. But you should always think of every possible case that can go wrong. Like in this example, what if you give more than 1 arguments here? Obviously, it will have no impact here, but you can always raise an error or print a message that says "This method/program accepts only 1 argument".
Do not change for now in any of the program in this PR, just a good practice.